cicd/pipelines/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: ASSEMBLY
+[id="using-tekton-chains-for-openshift-pipelines-supply-chain-security"]
+= Using Tekton Chains for OpenShift Pipelines supply chain security
+include::_attributes/common-attributes.adoc[]
+:context: using-tekton-chains-for-openshift-pipelines-supply-chain-security
+
+toc::[]
+
+:FeatureName: Tekton Chains
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+{tekton-chains} is a Kubernetes Custom Resource Definition (CRD) controller. You can use it to manage the supply chain security of the tasks and pipelines created using {pipelines-title}.
+
+By default, {tekton-chains} observes all task run executions in your {product-title} cluster. When the task runs complete, {tekton-chains} takes a snapshot of the task runs. It then converts the snapshot to one or more standard payload formats, and finally signs and stores all artifacts. 
+
+To capture information about task runs, {tekton-chains} uses the `Result` and `PipelineResource` objects. When the objects are unavailable, {tekton-chains} the URLs and qualified digests of the OCI images.
```

Comment:
also, about `PipelineResource` we should mention that it's deprecated and will be moved sometime in the future, so users should prefer to use `Result` instead.

Reply:
@concaf - Would you suggest any alternating phrasing? Or any specific portion that's unclear?